server/odc-service/src/main/java/com/oceanbase/odc/service/pldebug/session/AbstractDebugSession.java

```diff
@@ -190,11 +193,12 @@ protected void enableDbmsOutput(Statement statement) {
     static class DebugDataSource extends SingleConnectionDataSource {
 
         private final List<String> initSqls;
-        private final SessionCreatedInitializer initializer;
+        private final List<ConnectionInitializer> initializers;
 
         public DebugDataSource(@NonNull ConnectionConfig connectionConfig, List<String> initSqls) {
             this.initSqls = initSqls;
-            this.initializer = new SessionCreatedInitializer(connectionConfig, true);
+            this.initializers = Arrays.asList(new DataSourceInitScriptInitializer(connectionConfig, true),
```

Comment:
we may add BackupInstanceInitializer first and add DataSourceInitScriptInitializer second

Reply:
Sure, Nice suggestion! I have moved all BackupInstanceInitializer before DataSourceInitScriptInitializer~